src/minimega/vm.go

```diff
@@ -512,3 +512,50 @@ func ParseVmState(s string) (VmState, error) {
 
 	return VM_ERROR, fmt.Errorf("invalid state: %v", s)
 }
+
+// Get the VM info from all hosts optionally applying column/row filters.
+// Returns a map with keys for the hostnames and values as the tabular data
+// from the host.
+func globalVmInfo(masks []string, filters []string) map[string][][]string {
```

Comment:
This still seems web specific. If so, we should move it to web.go. I also sort of see this as a wrapper to runGlobally("vm info") or whatever the syntax for that is. If it's reduced to that, we should eliminate it.